src/mugs.js

```diff
@@ -1464,6 +1464,22 @@ define([
                     return data.id && data.id.slice(data.id.lastIndexOf("/") + 1);
                 }
             },
+            labelItext: {
```

Comment:
Oh, interesting, I hadn't found the itext validation function. Reading that, I think this is a bug in that function, where `hasHumanReadableItext` only validates that the itext has at least one form, but it doesn't validate that there's actually text for the form. Added https://github.com/dimagi/Vellum/pull/703/commits/eccef134b8f472c7d7d8db0034229e3dbeac4d3f